.gitignore

```diff
@@ -1,3 +1,4 @@
+.DS_Store
```

Comment:
NIT: if you configure your global `${HOME}/.gitignore` then this won't be required

```
$ cat ~/.gitignore        
...
# Folder view configuration files
.DS_Store
Desktop.ini
...
```

Reply:
you are right.

```
echo ".DS_Store" >> ~/.gitignore  
git config --global core.excludesfile ~/.gitignore
```